### DISCUSSION TITLE
Catch unsaved translations before saving page

### DISCUSSION THREAD

**Discussion by enzedonline** [wagtail/wagtail-localize#359] [Ideas]
I'm loving what I'm seeing so far - so much better to work with than wagtail trans 😊

One thing I find - it's easy to forget to press save on the translation then lose it when you save the page. Would be great to catch any open trans dialogues and either offer to save all or bump back to the edit page.

**Comment by kaedroho** [comment]
Thanks @enzedonline!

This is a very common issue (seen it happen multiple times over calls now!) so will definitely get this sorted before the 1.0 release
